gnome-base/gnome-vfs/gnome-vfs-2.24.4-r5.ebuild

```diff
@@ -90,15 +90,6 @@ PATCHES=(
 	"${FILESDIR}"/${PN}-2.24.4-openssl-1.1.patch
 )
 
-src_prepare() {
-	epatch "${PATCHES[@]}"
-
-	sed -e "s/AM_CONFIG_HEADER/AC_CONFIG_HEADERS/" -i configure.in || die
```

Comment:
this might come back to haunt us again?

Reply:
Probably not, with no more releases and a single revdep in tree.

Reply:
so you're confident Automake won't remove the `AM_CONFIG_HEADER` in a future release (like they did previously)? I'm fine going with that, just something to be aware of.